scripts/main/view.js

```diff
@@ -2,18 +2,25 @@
  * @description Responsible to reflect data changes to the UI.
  */
 
-let view = {};
+const view = {};
 
 view.albums = {
+	/** @returns {void} */
 	init: function () {
 		multiselect.clearSelection();
 
 		view.albums.title();
 		view.albums.content.init();
 	},
 
+	/** @returns {void} */
 	title: function () {
 		if (lychee.landing_page_enable) {
+			// TODO: Fix this odd condition. Also see `lychee.setTitle` for the associated TODO.
+			// I don't understand the intention of this code. IMHO, the code
+			//  should always be the same no matter if the prefix of the
+			// title equals the default `"Lychee v4"` or if it is customized
+			// by the user
 			if (lychee.title !== "Lychee v4") {
 				lychee.setTitle(lychee.title, false);
 			} else {
```

Comment:
That was legacy with v3 behaviour.

Reply:
I assumed so. However, I have no idea how ' view.album.title` _should_ work in particular in combination with `lychee.setTitle`.

I simply was too lazy to check all invocations of `view.albums.title`, which values are passed to it and how the current outcome looks like. I assume that the whole method could be boiled down to one or two lines if `lychee.setTitle` would be adopted accordingly.